src/data/services/LmsApiService.js

```diff
@@ -203,6 +207,14 @@ class LmsApiService {
   static fetchEnterpriseCustomerCatalogs(enterpriseId) {
     return LmsApiService.apiClient().get(`${LmsApiService.enterpriseCustomerCatalogsUrl}?enterprise_customer=${enterpriseId}`);
   }
+
+  static fetchUserDetailsFromEmail(emails) {
+    return LmsApiService.apiClient().post(`${LmsApiService.accountsSearchEmailsUrl}`, emails);
+  }
```

Comment:
I am not seeing usage of this?

Reply:
I think it got added during rebase, I will remove it now.